src/Components/animation/AnimationEditor.js

```diff
@@ -50,14 +48,43 @@ class AnimationEditor extends React.Component {
   };
 
   onLayout = e => {
+    var cHeight = e.nativeEvent.layout.height;
```

Comment:
I don't know exactly what is the `c` for? Maybe rename the variable so that it is explicit ? (churrosHeight if `c` stands for churros)

Reply:
c was for current but not very explicit indeed.